registry/client/src/appRoutes/List.js

```diff
@@ -72,6 +73,11 @@ const ListFilter = ({ routerDomain, ...props }) => {
                     />
                 : null
             }
+            {
+                !props.filterValues.showSpecial
+                ? <TextInput label="Route prefix" source="routePrefix" alwaysOn resettable/>
```

Comment:
```suggestion
                ? <TextInput label="Route" source="route" alwaysOn resettable/>
```

Reply:
why do you use word "prefix"? I think label can be just "Route" and source="route"

Reply:
To emphasise that it is a prefix based search. Ex:
/foo maches: /foo, /foo/ & /foo/bar but doesn't match /test/foo